synapse/config/registration.py

```diff
@@ -32,6 +32,7 @@ def read_config(self, config):
             )
 
         self.registration_shared_secret = config.get("registration_shared_secret")
+        self.user_creation_max_duration = config.get("user_creation_max_duration")
```

Comment:
This should probably be `config["user_creation_max_duration"]` as the default is handled by the default config below, and we probably don't ever want `user_creation_max_duration` to be `None` rather than an int